test/services/base.service.test.js

```diff
@@ -147,6 +167,23 @@ describe('base.service', () => {
           );
         });
     });
+
+    test('test for meta.lastUpdated inclusion', async () => {
+      await supertest(server.app)
+        .put('/4_0_1/Patient/testPatient2')
+        .send(UPDATE_PATIENT_2)
+        .set('Accept', 'application/json+fhir')
+        .set('content-type', 'application/json+fhir')
+        .set('x-provenance', JSON.stringify(SINGLE_AGENT_PROVENANCE))
+        .expect(200)
+        .then(async () => {
+          const patientCollection = db.collection('Patient');
+          const retrievedPatient = await patientCollection.findOne({ id: UPDATE_PATIENT_2.id });
+          expect(retrievedPatient.meta.lastUpdated).toBeDefined();
+          expect(new Date(retrievedPatient.meta.lastUpdated) > new Date(testPatient2.meta.lastUpdated));
```

Comment:
Sadly this assertion will always pass regardless of the value inside the parens here, since the jest expression `expect(true)` or `expect(false)` will never throw an error since there isn't a right side of the assertion :/ confusing I know.

We should add a right side here to make sure that the statement inside the `expect` evaluates to true.

```suggestion
          expect(new Date(retrievedPatient.meta.lastUpdated) > new Date(testPatient2.meta.lastUpdated)).toBe(true);
```